### ISSUE TITLE
Accessibility - Video Player - Focus order keyboard issues

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/ibm-dotcom-library#1770]
## Detailed description

Issues :- 
1.Picture in picture button
With tab/shift-tab focus is not going to the "Select region".

2.focus is going to non interactive elements
Focus is going to non interactive elements

3. Pressing esc key we are able to exit the full screen but focus is not set on the triggering element

Expected -
1.Focus should go to each and every interactive element and it should be operable by keyboard
2.Focus should only go to the interactive elements.
3. After exit the full screen with esc key focus should set on the triggering element which is "full screen button".

**Comment by RobertaJHahn** [comment]
Moving this issue to sprint 20-08.  The dev work has to be done during the first week of the sprint to include it in release v1.6.0.

**Comment by RaphaelAmadeu** [comment]
This bug is happening on the embed player, so we can't fix it on our end,
We can close this issue,
CC: @ljcarot @RobertaJHahn @jeffchew
